docs/process/teams.md

```diff
@@ -0,0 +1,13 @@
+These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.
+
+| Team | Purpose |
+|:--|:--|
+| `@desktop/analytics` | Analyzing the data we use for gaining insight into Desktop and its users |
+| `@desktop/maintainers` | The maintainers of GitHub Desktop |
+| `@desktop/comrades` | Avenue for community members to get more involved in the Desktop project |
+| `@desktop/docs` | Responsible for creating documentation Desktop users can consult when needed |
+| `@desktop/engineering` | Responsible for implementing Desktop's new features, fixing bugs, and all other developement tasks |
```

Comment:
in light of the recent discussion https://github.com/desktop/desktop/issues/5486, maybe it's helpful to call out `reviewing and merging PRs` also?